src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/Helper.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace System.Net.Http.Json
+{
+    internal static class Helper
+    {
+        internal static MediaTypeHeaderValue GetDefaultMediaType() => new("application/json") { CharSet = "utf-8" };
```

Comment:
Why change from the property to a method here?

Reply:
A method seemed more appropriate since a new instance is returned each time (doing non-trivial work).